properties/GirCore.Testing.props

```diff
@@ -1,14 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Project>
-  <PropertyGroup>
-    <IsPackable>false</IsPackable>
```

Comment:
I think the removal of `IsPackable` is missing in "Tests/Generation/Directory.Build.props".